cmake/GFlags.cmake.in

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(gflags-download NONE)
+
+# Enable ExternalProject CMake module
+include(ExternalProject)
+
+option(ALLOW_DOWNLOADING_GFLAGS "If gflags src tree is not found in location specified by GFLAGS_PATH, do fetch the archive from internet" OFF)
+set(GFLAGS_PATH "/usr/src/gflags" CACHE PATH
+                    "Path to the gflags root tree. Should contain CMakeLists.txt in root.")
+
+# Download and install GFlags
+
+message(STATUS "Looking for GFlags sources")
+message(STATUS "Looking for GFlags sources in ${GFLAGS_PATH}")
+if(EXISTS "${GFLAGS_PATH}"            AND IS_DIRECTORY "${GFLAGS_PATH}"            AND EXISTS "${GFLAGS_PATH}/CMakeLists.txt")
+  message(STATUS "Found GFlags in ${GFLAGS_PATH}")
+
+  ExternalProject_Add(
+    gflags
+    PREFIX            "${CMAKE_BINARY_DIR}"
+    DOWNLOAD_DIR      "${CMAKE_BINARY_DIR}/download"
+    SOURCE_DIR        "${GFLAGS_PATH}" # use existing src dir.
+    BINARY_DIR        "${CMAKE_BINARY_DIR}/build"
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND     ""
+    INSTALL_COMMAND   ""
+    TEST_COMMAND      ""
+  )
+else()
+  if(NOT ALLOW_DOWNLOADING_GFLAGS)
+    message(SEND_ERROR "Did not find GFlags sources! Either pass correct path in GFLAGS_PATH, or enable BENCHMARK_DOWNLOAD_DEPENDENCIES, or disable BENCHMARK_ENABLE_GTEST_TESTS / BENCHMARK_ENABLE_TESTING.")
```

Comment:
this bit of the error that talks about disabling testing doesn't apply to gflags.